### PR TITLE
add responsibleai-vision package to responsible-ai-toolbox

### PR DESCRIPTION
**Pull request by imatiach-msft**
## Description

Added responsibleai-vision package to responsible-ai-toolbox.
The python backend for explaining computer vision models.

## Checklist

- [ ] I have added screenshots above for all UI changes.
- [ ] I have added e2e tests for all UI changes.
- [ ] Documentation was updated if it was needed.

### CODE REVIEW

File: responsibleai_vision/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+numpy>=1.17.2
```
Comment (romanlutz) [inline responsibleai_vision/requirements.txt:1]: in requirements dev you have an upper limit, do you want to be consistent?

**Review by romanlutz — APPROVED** [review]

File: responsibleai_vision/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Responsible AI Vision SDK for Python
+
+### This package has been tested with Python 3.6, 3.7, 3.8 and 3.9
+
+The Responsible AI Vision sdk enables users to analyze their machine learning models for computer vision in one API. Users will be able to analyze errors, explain the most important features, and understand their data using a single API.
```
[the remark this answers is not in the document]
Comment (imatiach-msft) [reply responsibleai_vision/README.md:5]: done!

File: responsibleai_vision/requirements.txt (hunk 1 above)
Comment (imatiach-msft) [reply responsibleai_vision/requirements.txt:1]: interesting, I see different versions all over the place here:
image

Comment (imatiach-msft) [reply responsibleai_vision/requirements.txt:1]: I just went ahead and removed the one in requirements-dev.txt, as we don't need a duplicate one I think...

**Pull request merged**